CMakeLists.txt

```diff
@@ -157,12 +157,25 @@ if(CMAKE_COMPILER_IS_MSVC)
 
     include(ProcessorCount)
     ProcessorCount(CPUCores)
-    set(MSVC_MP ${CPUCores} CACHE STRING "Number of simultaneously running compilers (0 = automatic detection by MSVC). See documentation of /MP flag.")
-    if(MSVC_MP EQUAL 0)
-      # MSVC_MP is 0 in case the information cannot be determined by ProcessorCount => fallback
-      add_compile_options("/MP")
-    elseif(MSVC_MP GREATER 1)
-      add_compile_options("/MP${MSVC_MP}")
+    if (CMAKE_VERSION VERSION_LESS 3.11.0)
```

Comment:
With this condition shouldn't the first branch have old code and `else` branch new code?

Reply:
Yep - fixed.